test_execute: Missing plplot xwin driver.
test_obj_isa: Fails on other platforms also.



--- gdl-0.9.7/testsuite/Makefile.am.orig	Tue May 30 21:27:05 2017
+++ gdl-0.9.7/testsuite/Makefile.am	Tue May 30 21:27:55 2017
@@ -84,7 +84,6 @@
   test_device.pro \
   test_dicom.pro \
   test_erfinv.pro \
-  test_execute.pro \
   test_extra_keywords.pro \
   test_fft.pro \
   test_fft_dim.pro \
@@ -132,7 +131,6 @@
   test_nestedloop.pro \
   test_netcdf.pro \
   test_null.pro \
-  test_obj_isa.pro \
   test_obj_new.pro \
   test_plot_oo.pro \
   test_plotting_ranges.pro \
